### DISCUSSION TITLE
IDM Extension doesn't download Videos like Youtube , SSL Connect Error 5

### DISCUSSION THREAD

**Discussion by rezaei** [hiddify/hiddify-app#1121] [General]
Hi, a very useful application for download youtube videos is IDM ( and its Browser Extension) , but  in connect through VPN, i get this Error  "SSL Connect Error 5 "   , can i Ask you to solve this problem for your vpn connections

**Comment by voicerecordist** [comment]
same problem
i get this error:
Cannot download this file.
Details:
SSL_connect, select error 0
